code/API_definitions/device-status.yaml

```diff
@@ -997,6 +963,25 @@ components:
             status: 503
             code: "UNAVAILABLE"
             message: "Service unavailable"
+    SubscriptionIdRequired:
+      description: Problem with the client request
+      content:
+        application/json:
+          schema:
+            $ref: "#/components/schemas/ErrorInfo"
+          examples:
+            Generic400:
+              summary: Schema validation failed
+              value:
+                status: 400
+                code: INVALID_ARGUMENT
+                message: "Client specified an invalid argument, request body or query param"
+            subscriptionIdRequired:
+              summary: subscription id is required
+              value:
+                status: 400
+                code: INVALID_ARGUMENT
+                message: "Expected property is missing: subscriptionId"
```

Comment:
1. If the type is a special error, then the description could be also more concrete. 
2. This special 400 error shouldn't have a Generic400 as example

Reply:
I'm not completely getting the special error comment. Since in CAMARA we are using examples to include specific errors, I chose to create a different schema containing both examples. So this way we have two schemas, one containing only Generic 400 for the endpoints that doesn't have subscriptionId and another one containing both. 

Is there a better way to do it without adding it inline? or do you mean that it should only have the specific error because the Generic doesn't apply?

What description do you think would be more appropriate?

Reply:
I thought that since we also have the Generic400 error, then this SubscriptionIdRequired is a specialized one, so it could have a 
- description like "SubscriptionId is missing" and
- it should only have one example, exactly this case when subscriptionid is missing:
  
          examples:
                status: 400
                code: INVALID_ARGUMENT
                message: "Expected property is missing: subscriptionId"